.github/actions/time-compilation/action.yaml

```diff
@@ -0,0 +1,43 @@
+name: Time Compilation
+description: Time the cargo compilation, outputting an HTML file.
+
+inputs:
+  target:
+    description: "Target triple, like `wasm32-unknown-unknown"
+    required: false
+  target_option:
+    description: "--target={target}"
+    required: false
+  use_cache:
+    required: true
+    type: boolean
+
+runs:
+  using: composite
+  steps:
+    - name: 🔗 Setup toolchain
+      uses: actions-rs/toolchain@v1
```

Comment:
actions-rs was deprecated recently. Folks migrate to https://github.com/dtolnay/rust-toolchain or (as rust-analyzer) just not using any action at all https://github.com/rust-lang/rust-analyzer/pull/12764/files

Reply:
Thanks! https://github.com/prql/prql/issues/862